README.md

```diff
@@ -3,6 +3,8 @@ API Server for CogniCity
 
 [![Build Status](https://travis-ci.org/urbanriskmap/cognicity-server.svg?branch=master)](https://travis-ci.org/urbanriskmap/cognicity-server)
 
+[![Coverage Status](https://coveralls.io/repos/github/urbanriskmap/cognicity-server/badge.svg?branch=server-object-refactor)](https://coveralls.io/github/urbanriskmap/cognicity-server?branch=server-object-refactor)
+
```

Comment:
yes. Ideally we'd do these updates to the DB just via the API, but that's on hold until we can mock jwt tokens.

also, yes perhaps we could set this up to use a local test db. i.e. run the integration tests the same way Travis does (with a clean schema instance "cognicity_test")?